listenbrainz/webserver/views/donor_api.py

```diff
@@ -38,3 +39,19 @@ def biggest_donors():
 
     donors = get_biggest_donors(meb_conn, db_conn, count, offset)
     return jsonify(donors)
+
+
+@donor_api_bp.route("/all-flairs", methods=["GET", "OPTIONS"])
+@crossdomain
+@ratelimit()
+def all_flairs():
+    """ Get flairs (including expired ones) for all users """
```

Comment:
I think we need more details in this endpoint documentation. What does this endpoint return? Does the caller need to care about expired flairs?

Reply:
fixed.